website/content/en/docs/scorecard-alpha/_index.md

```diff
@@ -0,0 +1,222 @@
+  entrypoint:
+  - scorecard-test
+  - basic-check-spec
+  labels:
+    suite: basic
+    test: basic-check-spec-test
+  description: check the spec test
+- name: "olm-bundle-validation"
+  image: quay.io/operator-framework/scorecard-test:dev
+  entrypoint:
+  - scorecard-test
+  - olm-bundle-validation
+  labels:
+    suite: olm
+    test: olm-bundle-validation-test
+  description: validate the bundle test
+```
+
+The configuration file defines each test that scorecard can execute.  
+The following fields of the scorecard configuration file define the 
+test as follows:
+ 
+ * name - the human readable name you use to describe the test, 
+however, the actual name printed by scorecard when it executes the 
+test is defined within the test itself
+ * image - the test container image name that implements a test 
+ * entrypoint - the command and arguments that are invoked in the 
+test image to execute a test
+ * labels - user defined labels that allow for test selection using 
+the scorecard CLI `--selector` flag
+ * description - the human readable description for the test, however, 
+the description printed by scorecard on an executed test is the 
+description provided by the test implementation itself
+
+### Command Args
+
+The scorecard command has the following syntax:
+```
+operator-sdk alpha scorecard [bundle path] | [bundle image name] [flags]
+```
+
+The scorecard requires a positional argument that holds either the
+on-disk path to your operator bundle or the name of a bundle image.
+
+Scorecard flags include the following:
+
+| Flag        | Type   | Description   |
+| --------    | -------- | -------- |
+| `--config`  | string | path to config file (default `<bundle directory>/tests/scorecard/config.yaml`; file type and extension must be `.yaml`). If a config file is not provided and a config file is not found at the default location, the scorecard will exit with an error. |
+| `--kubeconfig`, `-o`  | string |  path to kubeconfig used by the scorecard to execute tests, if not supplied, the KUBECONFIG environment variable or $HOME/.kube/config path is assumed, if those are not found an in-cluster configuraiton is assumed. |
+| `--list`, `-L`  | bool |  if true, only print the test names that would be run based on selector filtering. |
+| `--namespace`, `-n`  | string |  the namespace for scorecard to use when executing test Pods |
+| `--output`, `-o`  | string | output format. Valid options are: `text` and `json`. The default format is `text`, which is designed to be a simpler human readable format.|
+| `--selector`, `-l`  | string |  the label selector to filter tests on. |
+| `--service-account`, `-s`  | string |  the service account for scorecard to use when creating test Pods.|
+| `--skip-cleanup`, `-x`  | boolean |  disable deleting test Pods and ConfigMaps generated from executing scorecard tests, useful for debugging.|
+| `--wait-time`, `-w`  | int |  seconds to wait for tests to complete.  Example is 35s.|
+
+## Tests Performed
```

Comment:
I think that the standard for doc titles are "action/ing". E.g Understanding tests performed

Reply:
hmmn, not sure that standard applies in all cases, but I updated one of these to be clearer.